sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -458,7 +473,14 @@ def test_linear_model_sample_weights_normalize_in_pipeline(
     # linear estimator with built-in feature normalization
     reg_with_normalize = estimator(normalize=True, fit_intercept=True,
                                    **params)
-    reg_with_normalize.fit(X_train, y_train, sample_weight=sample_weight)
+
+    if model_name in ['Lasso', 'ElasticNet']:
+        ctxmgr = pytest.warns(ConvergenceWarning)
```

Comment:
Apparently one of them is not raising the `ConvergenceWarning`.

Reply:
Really? I just tried both alone, and in both cases new warnings appear.

Reply:
Ahh indeed, in some of the CI runs it does indeed not error out ...

I guess that answers my other quesion that one should use `with ignore_warnings(category=ConvergenceWarning):`